homeassistant/components/apple_tv/__init__.py

```diff
@@ -50,16 +41,25 @@ def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     return value if isinstance(value, list) else [value]
 
 
+CREDENTIALS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_CREDENTIALS_MRP): cv.string,
+        vol.Optional(CONF_CREDENTIALS_DMAP): cv.string,
+        vol.Optional(CONF_CREDENTIALS_AIRPLAY): cv.string,
+    }
+)
+
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.All(
             ensure_list,
             [
                 vol.Schema(
                     {
-                        vol.Required(CONF_HOST): cv.string,
-                        vol.Required(CONF_LOGIN_ID): cv.string,
-                        vol.Optional(CONF_CREDENTIALS): cv.string,
+                        vol.Required(CONF_ADDRESS): cv.string,
+                        vol.Required(CONF_IDENTIFIER): cv.string,
+                        vol.Required(CONF_PROTOCOL): vol.In(["DMAP", "MRP"]),
```

Comment:
Please create constants for these two protocols in const.py and use them here.

Reply:
I will change to use name from definition in pyatv enum, e.g. Protocol.DMAP.name and Protocol.MRP.name.

Reply:
I think it is preferred to declare these constants instead of importing them from the library because it makes it easier to read the code without further knowledge of the library.

Reply:
Ok, sure! So maybe `PROTOCOL_DMP = "DMAP"` and corresponding for MRP in `const.py` then?

Reply:
Yes, indeed!